cmake/template/DLAFConfig.cmake.in

```diff
@@ -20,6 +20,7 @@ list(PREPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR})
 # ===== VARIABLES
 set(DLAF_WITH_OPENMP @DLAF_WITH_OPENMP@)
 set(DLAF_WITH_MKL @DLAF_WITH_MKL@)
+set(DLAF_WITH_MKL @DLAF_WITH_MKL_LEGACY@)
```

Comment:
Does the `DLAF_WITH_MKL` section not need updating below?

Reply:
ops...that's a misspell. good catch.

```suggestion
set(DLAF_WITH_MKL_LEGACY @DLAF_WITH_MKL_LEGACY@)
```

Reply:
Also that :sweat_smile: